File: src/main/java/dk/fitfit/solitaire/LaunchFX.java
package dk.fitfit.solitaire;

import javafx.application.Application;
import javafx.scene.Scene;
import javafx.scene.control.Label;
import javafx.scene.layout.StackPane;
import javafx.stage.Stage;

public class LaunchFX extends Application {
    @Override
    public void start(Stage stage) {
        new FXApplication().start(stage);
    }

    public static void main(String[] args) {
        launch();
    }
}

File: src/main/kotlin/dk/fitfit/solitaire/LaunchFX.kt
package dk.fitfit.solitaire

import javafx.application.Application
import javafx.stage.Stage

class LaunchFX : Application() {

    override fun start(stage: Stage) {
        val scene = DaggerMainComponent.builder()
            .build()
            .solitaireSceneFactory()
            .create()
        stage.scene = scene
        stage.show()
    }
}
